Master.py

```diff
@@ -822,9 +784,14 @@ def reporting(paths, param, tech):
 
     # Define sampling for sorted lists
     sampling = param["report_sampling"]
-
+    status = 0
     # Loop over each region
     for reg in range(0, nRegions):
+
+        # Display Progress
+        status += 1
```

Comment:
how does this work?
isn't status = reg +1?

Reply:
Status is just a counter, I compute what is the total number and count to it.  You are right here I could use reg +1, but I'd like to keep using status as it is standard in other parts of the script as well as the generate_model script